src/control/common/storage/commands.go

```diff
@@ -23,7 +23,7 @@ const MsgStoragePrepareWarn = "Memory allocation goals for SCM will be changed a
 type StoragePrepareNvmeCmd struct {
 	PCIAllowList string `long:"pci-allow-list" description:"Whitespace separated list of PCI devices (by address) to be unbound from Kernel driver and used with SPDK (default is all PCI devices)."`
 	PCIBlockList string `long:"pci-block-list" description:"Whitespace separated list of PCI devices (by address) to be ignored when unbinding devices from Kernel driver to be used with SPDK (default is no PCI devices)."`
-	NrHugepages  int    `short:"p" long:"hugepages" description:"Number of hugepages to allocate (in MB) for use by SPDK (default 1024)"`
+	NrHugepages  int    `short:"p" long:"hugepages" description:"Number of hugepages to allocate (2M hugepages) for use by SPDK (default 1024)"`
```

Comment:
Hmm. I don't think that DAOS requires 2MB hugepages, so this description will be incorrect if the system is using 1GB hugepages (or some other size). I think the parenthetical should just be removed.

Reply:
done